### ISSUE TITLE
Search: Accessibility issue when screen-reader reads out search results

### ISSUE THREAD

**Issue by auroranil** [Semantic-Org/Semantic-UI-React#1578]
It would be great if the React integration for Semantic-UI is more screen-reader-friendly, but we understand that this issue may be out of scope. Feel free to close this issue if you think this is the case.

### Steps
1. Install a screen-reader application
2. Go to http://react.semantic-ui.com/modules/search
3. Click on any search component to focus on it
4. Type something to bring up some search results
5. Press down (or up) arrow key to select a different search result

### Expected Result
The screen-reader application should be able to focus on each search result as you go up or down the search results. Once focused, the application will be able to read the text out loud from the search result.

### Actual Result
The screen-reader application is unable to focus on the currently active search result, and consequently is unable to read out the text.

### Version
0.67.2

**Comment by levithomason** [comment]
We've had some contributions to make this library more accessible and would love more.  You can reference these other PRs for guidance:

#1006 fix(Dropdown): added role/aria attributes to Dropdown and DropdownItem 
#1002 fix(Icon): added aria-hidden attribute to icon  ready for review
#1001 fix(List): Added aria roles for list/list-item  ready for review
#892 feat(Rating): Rating now keyboard navigable and visually impaired accessible needs author feedback

**Comment by thenorthstarblues** [comment]
Hi all! I'm looking to make my first contribution to the project. Can I pick up this issue?

**Comment by levithomason** [comment]
@thenorthstarblues please do, we'd be thrilled to have you. You can find helpful setup info in the CONTRIBUTING.md in the .github directory.

**Comment by thenorthstarblues** [comment]
Sounds good! Thank you.

**Comment by thenorthstarblues** [comment]
@levithomason I did a quick accessibility check on the doc site (using http://wave.webaim.org/) to see what other things might come up. There are a couple of small things like logos missing alt text. The screen reader issue might take a little bit, but I was wondering if it would be helpful to work on these other accessibility-related issues as well? Should I open new issues for them here? Let me know what you think is best...thanks!

**Comment by levithomason** [comment]
Thanks to @rijk, we have improved how we handle stale issues, see #2761.   Reopening.

**Comment by aasimsayani** [comment]
Hello all, is this still an issue that is open for contribution? Already looking over the CONTRIBUTING.md.  @levithomason Are there any other features or aspects of the project that need to be made accessible? I would be more than happy to contribute and assist with that. Just wanted some direction before diving in! Thanks!
